packages/generator-clay/app/templates/src/_Boilerplate.js

```diff
@@ -0,0 +1,33 @@
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import templates from './<%= componentName %>.soy.js';
+
+/**
+ * Metal <%= componentName %> component.
+ */
+class <%= componentName %> extends Component {
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+<%= componentName %>.STATE = {
+  /**
+	 * Id to be applied to the element.
+	 * @instance
+	 * @memberof <%= componentName %>
+	 * @type {?string|undefined}
```

Comment:
We don't need to specify that is `undefined` if we already mark it as optional with `?`. So `@type {?string}`